Generalize package name for the project (Closes #98)

szareiangm opened the pull request:
Just a package rename as we discussed in #98 ✋ 

CC: @Nafid

BenFradet commented on core/src/main/scala/com.snowplowanalytics.stream/loader/ElasticsearchPipeline.scala:
```diff
@@ -17,23 +17,19 @@
 
-package com.snowplowanalytics
-package elasticsearch.loader
+package com.snowplowanalytics.stream.loader
```
I don't think this needs modifying, it simplifies the snowplow imports

BenFradet commented on core/src/main/scala/com.snowplowanalytics.stream/loader/ElasticsearchSinkApp.scala:
```diff
@@ -52,7 +52,7 @@ trait ElasticsearchSinkApp {
     case class FileConfig(config: File = new File("."))
     val parser = new scopt.OptionParser[FileConfig](projectName) {
-      head(projectName, generated.Settings.version)
+      head(projectName, com.snowplowanalytics.stream.loader.generated.Settings.version)
```
this shouldn't need to be added

szareiangm replied:
for some reason my code doesn't compile without it.  Maybe some sbt settings with the new laptop.

BenFradet replied:
mmh weird

BenFradet commented on core/src/main/scala/com.snowplowanalytics.stream/loader/KinesisElasticsearchEmitter.scala:
```diff
@@ -17,15 +17,11 @@
 
-package com.snowplowanalytics
-package elasticsearch.loader
+package com.snowplowanalytics.stream.loader
```
same thing here let's leave it as is

BenFradet commented on core/src/main/scala/com.snowplowanalytics.stream/loader/KinesisSourceExecutor.scala:
```diff
@@ -17,8 +17,7 @@
 
-package com.snowplowanalytics
-package elasticsearch.loader
+package com.snowplowanalytics.stream.loader
```
same

BenFradet commented on core/src/main/scala/com.snowplowanalytics.stream/loader/SnowplowTracking.scala:
```diff
@@ -55,7 +55,7 @@ object SnowplowTracking {
     val method = config.method
     val emitter = AsyncEmitter.createAndStart(endpoint, Some(port), config.ssl.getOrElse(false), None)
-    new Tracker(List(emitter), generated.Settings.name, appName)
+    new Tracker(List(emitter), com.snowplowanalytics.stream.loader.generated.Settings.name, appName)
```
shouldn't be needed

BenFradet commented on core/src/main/scala/com.snowplowanalytics.stream/loader/SnowplowTracking.scala:
```diff
@@ -118,7 +118,7 @@ object SnowplowTracking {
     tracker.trackSelfDescribingEvent(SelfDescribingData(
       SchemaKey("com.snowplowanalytics.monitoring.kinesis", "app_initialized", "jsonschema", SchemaVer.Full(1,0,0)),
-      generated.Settings.name
+      com.snowplowanalytics.stream.loader.generated.Settings.name
```
same

BenFradet commented on core/src/test/scala/com.snowplowanalytics.stream.loader/BadEventTransformerSpec.scala:
```diff
@@ -0,0 +1,39 @@
+    }
+  }
+
+}
```
can we have the specs in different commits?

szareiangm commented:
@BenFradet I addressed all of your comments. Except having specs in different commit? I think my changes for them are the same as this issue. I only changed their package name. 
But I have a small change that I want if you could please ignore it for separate commit 😃  we missed Travis config file last time when I updated Scala version. I changed it here.

BenFradet commented:
ah weird I saw those specs as added code on github, merging thanks :+1:

snowplowcla commented:
@szareiangm has signed the Software Grant and Corporate Contributor License Agreement
